routes/error-tracker.js

```diff
@@ -209,11 +212,17 @@ function getHandler(req, res) {
       },
     },
   };
-  // get authentication context for logging
-  const loggingClient = logging({
-    projectId: appEngineProjectId,
-  });
-  const log = loggingClient.log(logName);
+  if (params.debug === '1' ) {
+    res.set('Content-Type', 'application/json; charset=ISO-8859-1');
+    res.status(statusCodes.OK).send(
+        JSON.stringify({
+          message: 'OK\n',
+          event: event,
+          throttleRate: throttleRate,
+        })).end();
```

Comment:
[The node documentation insist's that we do](https://nodejs.org/api/http.html#http_response_end_data_encoding_callback)